### ISSUE TITLE
Cannot install with pip3 install lasagne

### ISSUE THREAD

**Issue by thepunitsingh** [Lasagne/Lasagne#703]
When trying to install with `pip3 install lasagne` it gives the following error and do not install

```
Collecting lasagne
  Using cached Lasagne-0.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-pesa44au/lasagne/setup.py", line 10, in <module>
        CHANGES = open(os.path.join(here, 'CHANGES.rst')).read()
      File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 243: ordinal not in range(128)
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-pesa44au/lasagne/
```

**Comment by starcolon** [comment]
I found exactly the same issue on an Ubuntu machine.

**Comment by f0k** [comment]
Oh, sorry, seems I overlooked this in June. This is a duplicate of #463, fixed in #562. Since we haven't published a new release on PyPI since then, you will need to install from github:
http://lasagne.readthedocs.io/en/latest/user/installation.html#bleeding-edge-version
